data/campaigns/fri05.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,356 @@
+-- =========================
+-- Some formating functions
+-- =========================
+
+include "scripting/richtext_scenarios.lua"
+
+-- TODO(Nordfriese): Need new images for them all
+function claus(title, text)
+   return speech ("map:claus.png", "1CC200", title, text)
+end
+function henneke(title, text)
+   return speech ("map:henneke.png", "F7FF00", title, text)
+end
+function reebaud(title, text)
+   return speech ("map:reebaud.png", "55BB55", title, text)
+end
+function ketelsen(title, text)
+   return speech ("map:ketelsen.png", "4455CC", title, text)
+end
```

Comment:
for the speech boxes I have 3 comments:
1. all images have a blueish border, which makes it har to distinguish the speaker. this is worsened by the images used
2. hennecke and ketelsen have the same image.
3. in the texts or headlines it is not given who is the current speaker.   

so please give them borders in the color of their heading colour and put the name of them in the headline (e.g. diary of claus lembeck instead of introduction or something similar)